.github/workflows/ecr-build.yml

```
@@ -0,0 +1,64 @@
+  AWS_ACCOUNT_ID: ${{ vars.AWS_ACCOUNT_ID }}
+  ECR_REPO: ${{ vars.AWS_ACCOUNT_ID }}.dkr.ecr.${{ vars.AWS_REGION }}.amazonaws.com/genai
+  IS_PR: ${{ github.event_name == 'pull_request' }}
+  SHOULD_PUSH_TO_ECR: ${{ contains(github.event.pull_request.labels.*.name, 'ecr-push') || github.event_name == 'merge_group' }}
+  S3_CACHE_BUCKET: ${{ secrets.S3_CACHE_BUCKET }}
+
+jobs:
+  docker-build:
+    uses: ./.github/workflows/reusable-ecr.yml
+    with:
+      env: dev
+      aws-account-id: ${{ env.AWS_ACCOUNT_ID }}
```

actionlint (1.7.4)

context "env" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)